configs/benchmark-leo.json

```diff
@@ -54,7 +54,7 @@
     "crosstx": 30,
     "attacked_mode": 0,
     "peer_per_shard": 30,
-    "minpeer": 20,
+    "minpeer": 6,
```

Comment:
should it be 18 (30 * 0.66)? or this is limited by libp2p then it has to be 6 (10 physical nodes * 0.66)?

Reply:
you are right, physical node is only 10 per shard, thus minpeer has to be changed to 6 or lower. Or the bootstrap won't start. The minpeer is just collect ping message, which is one per node.